app/components/price-graph/price-graph.tsx

```diff
@@ -144,6 +149,8 @@ export const PriceGraph: ComponentType = ({
         return translate("PriceScreen.thisMonth")
       case GraphRange.ONE_YEAR:
         return translate("PriceScreen.thisYear")
+      case GraphRange.FIVE_YEARS:
+        return translate("PriceScreen.lastFiveYears")
```

Comment:
does this require update galoy-client first, right?

Reply:
@eimolina https://github.com/GaloyMoney/galoy-client/pull/50 was merged and published.. Please update package with the new version

Reply:
![image](https://user-images.githubusercontent.com/10089634/178063337-b20dc8de-ceb3-41a3-8c48-dd7936b1a327.png)

Reply:
I just updated the galoy-client library in the dependencies.